src/lib/Constants.ts

```diff
@@ -13,11 +13,11 @@ export const TX_VALIDITY_WINDOW = 120;
 // Labels
 export const LABEL_MAX_LENGTH = 63; // in bytes
 
-// Wallet
+// Accounts
 export const WALLET_BIP32_BASE_PATH_KEYGUARD = `m/44'/242'/0'/`;
-export const WALLET_DEFAULT_LABEL_KEYGUARD = 'Keyguard Wallet';
-export const WALLET_DEFAULT_LABEL_LEDGER = 'Ledger Wallet';
-export const WALLET_DEFAULT_LABEL_LEGACY = 'Legacy Wallet';
+export const WALLET_DEFAULT_LABEL_KEYGUARD = 'Keyguard Account';
+export const WALLET_DEFAULT_LABEL_LEDGER = 'Ledger Account';
+export const WALLET_DEFAULT_LABEL_LEGACY = 'Legacy Account';
 export const WALLET_MAX_ALLOWED_ACCOUNT_GAP = 20;
```

Comment:
Rename the constant variable names as well? Remember to rename the old `ACCOUNT_*` variables first, or you will not know which were the old and which are the new `ACCOUNT_*` variables.